lib/fieldhand/get_record_parser.rb

```diff
@@ -0,0 +1,23 @@
+require 'fieldhand/record'
+
+module Fieldhand
+  # A parser for GetRecord responses
+  #
+  # See https://www.openarchives.org/OAI/openarchivesprotocol.html#GetRecord
+  class GetRecordParser
```

Comment:
I've changed my mind: let's leave the specific response parsers as that means these are the _only_ objects that need to know what the path and object type of each response is.

If I pass these as arguments instead then the `Repository`, `Paginator` _and_ `ResponseParser` would all need to know those details.